docs/ivys.js

```diff
@@ -54,6 +76,8 @@ var emptyProductState = function (params) {
         tags: {},
         // Collection name
         collection: params.collection
+        // Should I also include the image in here? 
```

Comment:
If you add questions like this and then don't flag them in the PR, I almost certainly won't notice them. You can create the PR and then add comments like I'm doing now. That way it's obvious when I haven't answered something.